src/gsy_e/models/strategy/infinite_bus.py

```diff
@@ -71,6 +73,13 @@ def _set_global_market_maker_rate(self):
         elif self.energy_rate is not None:
             GlobalConfig.market_maker_rate = self.energy_rate
 
+    def _set_global_feed_in_tariff_rate(self):
+        if self.buying_rate_profile is not None:
+            GlobalConfig.FEED_IN_TARIFF = read_arbitrary_profile(
+                InputProfileTypes.IDENTITY, self.buying_rate_profile)
+        elif self.energy_buy_rate is not None:
+            GlobalConfig.FEED_IN_TARIFF = self.energy_buy_rate
```

Comment:
This is the actual change to set feed-in tariff as per buying rate of InfiniteBus.